packages/ui/src/app/pages/Forum/ForumCategory.tsx

```diff
@@ -113,4 +128,8 @@ export const ForumCategory = () => {
 
 const ModeratorsContainer = styled(Label)`
   align-items: center;
+  flex-direction: column;
+`
+const Flex = styled(PageHeaderWrapper)`
+  flex-direction: column;
```

Comment:
@gyroflaw here I meant that you should remove this `Flex` component and just add `flex-direction: column;` to the `PageHeaderWrapper` component directly.

Reply:
@thesan 
I think this might exist since `PageHeaderWrapper` is used in other components.
If we add flex-direction to the component directly, I think another used component's design would change.

Reply:
Yes `PageHeaderWrapper` is used in all pages but from looking through the app it looks to me like changing `flex-direction` to `column` shouldn't break anything.

Reply:
Okay, I will do. @thesan